lib/danger/request_sources/bitbucket_cloud_api.rb

```diff
@@ -105,6 +107,12 @@ def fetch_pr_from_branch(branch_name)
       end
 
       def fetch_access_token(environment)
+        access_token = environment["DANGER_BITBUCKETCLOUD_REPO_ACCESSTOKEN"]
```

Comment:
It is the same environment variable as https://github.com/danger/danger-js/blob/5edb114d07d57292c707bba1dd72b129edd2f44f/source/platforms/bitbucket_cloud/BitBucketCloudAPI.ts#L76